### ISSUE TITLE
Toggle to enable old sidebar behaviour when in Compact View

### ISSUE THREAD

**Issue by huds601** [FrankerFaceZ/FrankerFaceZ#1238]
**Is your feature request related to a problem? Please describe.**
Twitch recently pushed their new hover context menu live when you have the streamer sidebar collapsed. This requires you to hover over a streamer profile for approx. 1.5 seconds to see the stream information for that streamer (game, viewers etc.)

![image](https://user-images.githubusercontent.com/42800359/177727261-4718d41d-3c8a-4aea-ada4-7ddb2ba1136b.png)

**Describe the solution you'd like**
Make the vxp_left_nav_hover_logged_in option a permanent setting in FFZ to override the Twitch default and re-enable the old sidebar behaviour where the whole sidebar would slide out on hover.

![image](https://user-images.githubusercontent.com/42800359/177727201-6522af93-69d7-4283-a5b0-ffbef000d59b.png)

**Describe alternatives you've considered**
I have re-enabled the old behaviour using the Twitch Experiments in FFZ, but this does not persist between sessions.

![image](https://user-images.githubusercontent.com/42800359/177727334-ba7770d5-4ce0-45c5-9d6e-b8bda43a33d9.png)

**Additional context**
This is controlled by the vxp_left_nav_hover_logged_in experiment. Setting this to Treatment enables the old style slide-out sidebar.

**Comment by Tassader** [comment]
There was already issue #1229 about it.

But yes, this is annoying. The setting vxp_left_nav_hover_logged_in feels like workaround than anything else, AND it gets reset like every day

**Comment by Fogsight** [comment]
It seemed to have been fixed in #1185 but it's not saving settings again. 

Every time browser restarts have to open up Experiments menu, type the key string, locate it and set it. Over and over again.

**Comment by nogenem** [comment]
PLEASE, for love of God, can this be done? I'm having to set the vxp_left_nav_hover_logged_in every day and its quite annoying ;/

**Comment by Tassader** [comment]
I think the issue is that the cookie experiments is set to expire in 7 days so ... every 7 days you have to do the changes again

https://github.com/FrankerFaceZ/FrankerFaceZ/blob/155938f584d16efe39f5c348ce96389fbf02658a/src/experiments.js#L17

**Comment by BlastDamage** [comment]
Would love to see this implemented as an option so I don't have to toggle the experiment every few days

**Comment by nogenem** [comment]
I started using this script on "Tampermonkey" extension and didn't have a problem since...

```js
// ==UserScript==
// @name         Twitch - Set experimental cookie
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  Set experimental cookie
// @author       You
// @match        https://www.twitch.tv/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=twitch.tv
// @grant        none
// ==/UserScript==

(function() {
    'use strict';

    function setCookie(
     name/*: string*/,
     value/*: string = ''*/,
     days/*: number | false = false*/, // session length if not provided
     path/*: string = '/'*/, // provide an empty string '' to set for current path (managed by a browser)
     domain/*: string = ''*/,
     sameSite/*: 'none' | 'lax' | 'strict' = 'lax'*/, // required by Firefox
     isSecure/*?: boolean*/
    ) {
        let expires = '';
        if (days) {
            var date = new Date(
                Date.now() + days * 24 * 60 * 60 * 1000
            ).toUTCString();
            expires = '; expires=' + date;
        }
        var secure = isSecure || sameSite === 'none' ? `; Secure` : '';
        var domainStr = !!domain && domain.length > 0 ? `; Domain=${domain}` : '';
        var encodedValue = encodeURIComponent(value);
        document.cookie = `${name}=${encodedValue}${expires}; path=${path}; SameSite=${sameSite}${secure}${domainStr}`;
    }

    var expCookieMatch = document.cookie.match(/experiment_overrides=([^;]+);/);
    console.log('expCookieMatch', expCookieMatch);

    var shouldSetCookie = false;
    if(expCookieMatch && expCookieMatch[1]) {
        var expCookieStr = decodeURIComponent(expCookieMatch[1]);
        console.log('expCookieStr - decoded', expCookieStr);

        var expCookie = JSON.parse(expCookieStr);
        console.log('expCookie', expCookie);

        if(!expCookie.experiments || Object.keys(expCookie.experiments).length === 0) {
            shouldSetCookie = true;
        }
    } else {
        shouldSetCookie = true;
    }

    console.log('shouldSetCookie', shouldSetCookie);
    if(shouldSetCookie) {
        setCookie('experiment_overrides', '{"experiments":{"ce577266-68fa-40eb-8c07-5ae30488338f":"treatment"},"disabled":[]}', 180, '/', '.twitch.tv', 'none', true);
        //document.location.reload();
    }
})();
```

**Comment by BlastDamage** [comment]
> I started using this script on "Tampermonkey" extension and didn't have a problem since...

Thank you for this! While an official option would be nice, this solves the problem perfectly for the time being.

**Comment by alkaris2** [comment]
I agree with this `vxp_left_nav_hover_logged_in` SHOULD be a **Appearance > Layout** setting that can be set as default override. Because it's really annoying having to scroll all the way down to Experiments, type in `blah blah password whatever` and then Ctrl+F search for `vxp_left_nav_hover_logged_in` and set `treatment (weight:0)` every time. I don't know why you can't just have it as a regular toggle fix, because not being able to just automatically expand on hover over the sidebar when mouse rolls over it is really annoying.

**Comment by alkaris2** [comment]
I just notice that this is same issue referenced in #1184 so it's not just a handful of people complaining this isn't a layout option you can just toggle without having to go into the Experiments section to change it.

**Comment by funky-d** [comment]
My FFZ plugin updated today, and it appears the vxp_left_nav_hover_logged_in option has been removed from the experimental values. Why? I've been using this for years with no issue and agree with the OP that this should be included by default in the Appearance -> Layout -> Side Navigation section.

**Comment by Lordmau5** [comment]
Your FFZ plugin hasn't updated - Twitch has deprecated the experiment. Nothing FFZ can do about that, except for implementing all of it from scratch.

**Comment by random42** [comment]
Twitch is so braindead, I can't even click "show more" of my following now because of this. Can the old menu be implemented by the extension?
